fcsa_service_request-1.0.0-resolved.json

```diff
@@ -0,0 +1,567 @@
+{
+  "openapi": "3.0.0",
+  "info": {
+    "title": "CCT Customer Service Requests",
+    "description": "This section provides an overview of all aspects of the application programmers’ interface, including registration to use the interface, the various resources that constitute the interface and how these should be orchestrated into an application that will deliver consistent and reliable results.",
+    "contact": {
+      "email": "cct@cct.co.za"
+    },
+    "license": {
+      "name": "Apache 2.0",
+      "url": "http://www.apache.org/licenses/LICENSE-2.0.html"
+    },
+    "version": "2.1"
+  },
+  "servers": [
+    {
+      "url": "https://qaeservices1.capetown.gov.za/coct/api/zcur-guest/login",
+      "description": "FCSA Service Request API"
+    }
+  ],
+  "security": [
+    {},
+    {
+      "cookieAuth": []
+    }
+  ],
+  "tags": [
+    {
+      "name": "service_request_group",
+      "description": "This process shows the flow of actions in which the application takes for the creation of a service request. \n"
+    }
+  ],
+  "paths": {
+    "/coct/api/zcur-guest/login": {
+      "get": {
+        "tags": [
+          "service_request_group"
+        ],
+        "description": "Ths services is used to authenticate and retrieve the MYSAPSSO2 cookie or authentication token. \n",
+        "parameters": [
+          {
+            "name": "cookie",
+            "in": "query",
+            "required": true,
+            "style": "form",
+            "explode": true,
+            "schema": {
+              "type": "string",
+              "format": "uuid"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "headers": {
+              "Set-Cookie": {
+                "description": "Pass this cookie back in subsequent requests.\n",
+                "style": "simple",
+                "explode": false,
+                "schema": {
+                  "$ref": "#/components/schemas/Login_Schema"
+                }
+              }
+            }
+          }
+        },
+        "security": [
+          {
+            "cookieAuth": []
+          }
+        ]
+      }
+    },
+    "/coct/api/zsreq/session": {
+      "get": {
+        "tags": [
+          "service_request_group"
+        ],
+        "summary": "This operation retrieves a new session identifier for the X-Session header parameter.\n",
+        "parameters": [
+          {
+            "name": "cookie",
+            "in": "query",
+            "description": "This is the value of the public key issued  by the city to registered consumers.\n",
```

Comment:
"The MYSAPSSO2 authentication cookie. See /coct/api/zcur-guest/login path for details on how to retrieve it"